documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+        "YX", S=0, T=0, C=0
+    )
+    # NOTE: For datasets with 3D segmentations, you may need to flatten the data into
+    # 2D images. Typically, it's simplest to do so with a max projection, but may vary
+    # based on your data. Replace the above line with the following:
+    #
+    # segmentation_image = bioio.BioImage(frame_path).get_image_data("ZYX", S=0, T=0, C=0)
+    # segmentation_image = segmentation_image.max(axis=0)
+    #
+    # Remap the segmented so object IDs are unique across all timepoints.
+    (remapped_segmentations, _lut) = remap_segmented_image(
+        segmentation_image, frame_data, OBJECT_ID_COLUMN, INDEX_COLUMN
+    )
+    # Write the new segmentation image.
+    frame_prefix = "frame_"
+    frame_suffix = ".png"
+    writer.write_image(remapped_segmentations, frame_num, frame_prefix, frame_suffix)
+    frame_paths.append(frame_prefix + str(frame_num) + frame_suffix)
+
+writer.set_frame_paths(frame_paths)
+```
+
+#### 6. Write the dataset and any additional metadata
+
+```python
+# Define the metadata for this dataset.
+metadata = ColorizerMetadata(
+    name="Example dataset",
+    description="An example dataset for the Timelapse Feature Explorer.",
+    author="Jane Doe et al.",
+    dataset_version="v1.0",
+    # The width and height of the original segmentations, in any arbitrary units.
+    # This will control the scale bar in the viewer.
+    frame_width=100,
+    frame_height=100,
+    frame_units="nm",
+    # Time elapsed between each frame capture, in seconds.
+    frame_duration_sec=1,
+)
+
+# Write the final dataset
+writer.write_manifest(metadata=metadata)
+```
+
+Once the steps are run, the new dataset will be found in the `processed_dataset` directory.
+
+## 5. Viewing the dataset
+
+Now that the dataset is processed, we can view it in the Timelapse Feature Explorer!
+
+Our public release of Timelapse Feature Explorer is designed to load datasets hosted on a web server. To load **local datasets and files**, you'll need to run a **local instance** of the viewer. We'll cover steps for both options.
+
+### Viewing datasets via the web
+
+> **_NOTE:_** To use a dataset with our public build of Timelapse Feature Explorer, the dataset must be accessible using the HTTPS protocol (e.g., `https://example.com/your-dataset/`). If you need to use HTTP, run a local instance of the viewer.
+
+For this tutorial, you can load a pre-processed example copy of the dataset, which we've hosted on GitHub. You can access it at this URL: [https://raw.githubusercontent.com/allen-cell-animated/colorizer-data/main/documentation/getting_started_guide/processed_dataset](https://raw.githubusercontent.com/allen-cell-animated/colorizer-data/main/documentation/getting_started_guide/processed_dataset)
+
+If you have updated the dataset files or want to use a local dataset, skip to the next section ([Installing Timelapse Feature Explorer locally](#installing-timelapse-feature-explorer-locally)).
```

Comment:
Should I not show users how to do this, and force them to only learn to run a local instance?